test/visualize/test_visualize.py

```diff
@@ -1240,3 +1241,54 @@ def test_parameters_correlation_matrix(result_creation):
     result = result_creation()
 
     visualize.parameters_correlation_matrix(result)
+
+
+@close_fig
+def test_plot_ordinal_categories():
+    example_ordinal_yaml = (
+        Path(__file__).parent
+        / ".."
+        / ".."
+        / "doc"
+        / "example"
+        / "example_ordinal"
+        / "example_ordinal.yaml"
+    )
+    petab_problem = petab.Problem.from_yaml(example_ordinal_yaml)
+    # Set seed for reproducibility.
+    np.random.seed(0)
+    optimizer = pypesto.optimize.ScipyOptimizer(
+        method="L-BFGS-B", options={"maxiter": 10}
```

Comment:
For just testing that visualization does not produce an error, a single iteration would suffice, right?